ingestors/support/convert.py

```diff
@@ -59,6 +59,7 @@ def _document_to_pdf(self, file_path, entity):
                         bytes_written += len(chunk)
                         fh.write(chunk)
                     if bytes_written > 50:
+                        log.info("converted %s to PDF (attempt %)", entity, attempt)
```

Comment:
The build is complaining because ... I'm not entirely sure what's up with the formatting string, but feel free to use the fancy, newer f-string:

```suggestion
                        log.info(f"converted {entity} to PDF (attempt {attempt})")
```